README.md

```diff
@@ -49,12 +49,9 @@ crystal spec
 And the code should be in `/src` and the tests in `/spec`
 
 ### Deployment
-You can easily deploy this. Just clone this to your computer and run `docker build .` and deploy that container to your server. Kindmetrics use all lucky framework environment variables which can be seen here: https://luckyframework.org/guides/deploying/ubuntu#creating-a-systemd-unit-file-for-your-app
-
-Kindmetrics is dependent on some SMTP mail server and postgresql. Be sure to have those prepared.
+You can find the deployment instructions in [deployment.md file](https://github.com/kindmetrics/kindmetrics/blob/master/deployment.md)
```

Comment:
Just a detail, but if you use a relative path (`[deployment.md file](/deployment.md)`) you can easily migrate to Git(Lab/tea/whatever) at a later point and still have working links. :wink:

Reply:
Good point. will fix that :)